scripts/wsrep_sst_rsync.sh

```diff
@@ -178,6 +178,21 @@ FILTER=(-f '- /lost+found'
 if [ "$WSREP_SST_OPT_ROLE" = "donor" ]
 then
 
+||||||| merged common ancestors
```

Comment:
Version Control Marker will cause rsync tests to fail. Please remove this.


>2022-01-19T07:14:07.489217Z 0 [Note] WSREP: Initiating SST/IST transfer on JOINER side (wsrep_sst_rsync --role 'joiner' --address '127.0.0.1:13009' --datadir '/home/venki/work/pxc/review/nitendra/bld/mysql-test/var/mysqld.2/data/' --defaults-file '/home/venki/work/pxc/review/nitendra/bld/mysql-test/var/my.cnf' --defaults-group-suffix '.2' --parent '39359' --mysqld-version '5.7.36-39'   '' ) 
/home/venki/work/pxc/review/nitendra/bld/scripts/wsrep_sst_rsync: line 181: syntax error near unexpected token `||'
>2022-01-19T07:14:07.661565Z 0 [ERROR] WSREP: Failed to read 'ready <addr>' from: wsrep_sst_rsync --role 'joiner' --address '127.0.0.1:13009' --datadir '/home/venki/work/pxc/review/nitendra/bld/mysql-test/var/mysqld.2/data/' --defaults-file '/home/venki/work/pxc/review/nitendra/bld/mysql-test/var/my.cnf' --defaults-group-suffix '.2' --parent '39359' --mysqld-version '5.7.36-39'   '' 
	Read: '(null)'